front/src/utils/strings.ts

```diff
@@ -34,3 +36,11 @@ export function snakeToCamel(str: string) {
 }
 
 export const emptyStringRegex = /^\s+$/;
+
+export function joinTranslationList(translationList: string, item: string): string {
+  return `${translationList ? `${translationList}.` : ''}${item}`;
+}
+
+export function isParamNull(str: string | null): string {
+  return !isNull(str) ? ` ${str}` : ' N/C';
```

Comment:
better to use this, it's more performant :)
```suggestion
  return str !== null ? ` ${str}` : ' N/C';
```